gno.land/cmd/gnoland/README.md

```diff
@@ -6,8 +6,88 @@
     $> cd ./gno/gno.land
     $> make install.gnoland
 
-## Run `gnoland` full node
+## Option 1: Run `gnoland` full node for local development
 
     $> gnoland start
 
 Afterward, you can interact with [`gnokey`](../gnokey) or launch a [`gnoweb`](../gnoweb) interface.
+
+
+## Option 2: Run a node and sync with a Proof of Authority (POA) network
+
+ - gnoland init
+
+ - Download genesis.json from a trusted source and save it to the root-dir/config directory.
+
+ - Get the peer's node id from the node you trust.
+
+ - Start the node
+
+       $> gnoland start --persistent "node_id@peer_ip_address:port" or add the persistent_peers value in the ./testdir/config/config.toml
+
+## Option 3: Run a node as a Proof of Authority validator starting from genesis state
+
+- Initialize the config and key files.
+
+      $> gnoland init
+
+- Return the node info; we will need it to add to validator info in the genesis.json
+
+      $> gnoland node
+
+      Address: "g14t47gv3v2z3pc23g3zr39mnc99w2cplp0jhqvv"
+      Pubkey: "E5IFULgXFdS49ILgvPmO3/8chuSWfbqw3zYXaNEP+60="
+
+- Download genesis.json from a trusted source and save it to the root-dir/config directory.
+
+- Add your validator to the genesis file.
```

Comment:
If we want to retain the POA title in this PR, which I find confusing, I suggest revamping the example. Instead of just adding yourself as a validator, the example should emphasize coordinating with others and running the genesis validator add multiple times.

The current explanation is not clear. A possible improvement is to instruct validators to add their node ID to the shared genesis file (e.g., on a GitHub repo) and ensure they start with persistent node IDs for connectivity with the updated genesis file.

However, it's important to include a prominent warning that this approach is not actually PoA or Po something. It's simply a genesis with peers and no proof of anything except "being in the genesis."

Reply:
This is the `gentx` system for validators to be able to add themself inside the genesis.json
cf: https://github.com/gnolang/gno/pull/1102